.gitignore

```diff
@@ -12,6 +12,7 @@ coverage.cov
 **/obj/
 coverage/
 venv/
+.envrc
```

Comment:
I can remove this if we want - it eases use of `direnv` to have this in here, though.

Reply:
It's fine